src/Components/Coord.js

```diff
@@ -40,7 +40,7 @@ function Coord(props) {
           </Form.Text>
         </Form.Group>
 
-        <Button variant="primary" type="submit">
+        <Button variant="primary" type="button" onClick={props.onSubmit}>
```

Comment:
@Teatoller This didn't need to be a `submit` type button, just a `button` type. The fix was to add an event listener for this button that triggered a call to the API endpoint with the updated geolocations. Please note, if you see your browser refreshing, you haven't correctly implemented your react application as a Single Page Application.

Reply:
@bettblake08 Thank for this insight and assisting me to troubleshoot it.

Regards
Steven Ennis